### DISCUSSION TITLE
Lora transmission happens only in SF7 and SF8

### DISCUSSION THREAD

**Discussion by kvkc97** [Lora-net/LoRaMac-node#1248] [Q&A]
I am working on LR1110 in IN865 in ABP mode. Data transmission is successful in SF7 and SF8 but not in other spreading factors. The size of the packet is 17 bytes and adr is turned off. Frequency of transmission is 865062500. What can be issue for other spreading factors?

**Comment by mluis1** [comment]
Unfortunately you do not provide enough information in order to help you.
1. What branch or tag of this repository is your code based on?
2. What HW platform is being targeted?
3. Have you modified any file under `mac` directory?

Why you mention a single frequency?

A LoRaWAN end-device uses different frequencies for the uplinks.
In case of IN865 it uses by default at least 3 frequencies (865.0625 MHz, 865.4025 MHz and 865.985 MHz)

Are you using a real LoRaWAN gateway? (SX130x 8 channels based gateway)

How have you instructed the end-device to use a single frequency?

This project implementation of the IN865 region has been LoRaWAN pre-certified using the LCTT tool. While running these tests all region supported data rates are exercised.

I don't see any valid reason why other data rates wouldn't work.

**Comment by mluis1** [comment]
For information we have updated the LR1110 drivers to the latest available version under [v5.0.0-branch](https://github.com/Lora-net/LoRaMac-node/tree/v5.0.0-branch)
